AllReadyApp/Web-App/AllReady/Views/Campaign/ManageCampaign.cshtml

```diff
@@ -0,0 +1,102 @@
+                </div>
+                <div class="form-group">
+                    <button type="button" class="btn btn-default btn-sm" data-bind="css: { active: campaigns.showOld }, click: campaigns.showOld.toggle">
+                        <i class="fa" data-bind="css: { 'fa-square-o': !campaigns.showOld(), 'fa-check-square-o': campaigns.showOld }"></i>
+                        Include Completed Campaigns
+                    </button>
+                    <span data-bind="text: '(' + total + ')' " ></span>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+
+<div class="row">
+    <div class="hide text-center" data-bind="css: { hide: campaigns.filtered().length }">
+        <div class="alert alert-warning" role="alert">
+            <strong>No matching campaigns.</strong>
+        </div>
+    </div>
+    <div class="col-md-12">
+        <div id="campaignGrid" class="table-responsive">
+            <table class="table" data-bind="css: { hide: !campaigns.filtered().length }">
+                <tr>
+                    <th class="col-md-3"><span title="Name of the campaign">Title</span></th>
+                    <th class="col-md-3"><span title="Organizer of the campaign">Organizer</span></th>
+                    <th class="col-md-4"><span title="Description of the campaign">Description</span></th>
+                    <th class="col-md-2"><span title="Date of the campaign">Dates</span></th>
+                </tr>
+                <!-- ko foreach: campaigns.filtered -->
+                <tr>
+                    <td>
+                        <a data-bind="attr: { href: '/Campaign/' + Id }, text: Name"></a>
+                        <i data-bind="visible: Featured" class="fa fa-star star-highlight" title="Featured" aria-hidden="true"></i>
+                    </td>
+                    <td>
+                        <span data-bind="text: ManagingOrganizationName"></span>
+                        <div data-bind="visible:ManagingOrganizationLogo != '' || ManagingOrganizationLogo != null"><br/><img width="250" data-bind="attr:{src: ManagingOrganizationLogo}" /></div>
+                    </td>
+                    <td>
+                        <span data-bind="text: Description"></span>
+                    </td>
+                    <td>
+                        <span data-bind="text: displayDate()"></span>
+                    </td>
+                </tr>
+                <!-- /ko -->
+            </table>
+
+            <div data-bind="css: { hide: !featuredCampaigns }">
+                <i class="fa fa-star star-highlight" aria-hidden="true"></i>
+                Featured campaign(s)
+            </div>
+        </div>
+    </div>
+</div>
+
+@section scripts {
+    <script>
+        var modelCampaigns = @Json.Serialize(Model);
+    </script>
+    <script src="~/js/campaigns.js"></script>
+}
```

Comment:
I don't know if this page needs all the functionality of the Campaign Index, but when I create a new view model, with its property reduced, the ko code doesn't work, I don't know how to use ko, I just previously assumed because the view model is the same, the code will work (feeling guilty about it), I should either remove the ko code, or learn how to use it in this case, I should check this with @stevejgordon.